src/CoreBundle/private/library/classes/TFullGroupTable/TFullGroupTable.class.php

```diff
@@ -868,6 +868,19 @@ protected function _BuildFilterSection()
 
             $filterContent .= '<div class="form-group mr-2 typeahead-relative">';
 
+            $filterContent .= '<script>
+                function resetPagingAndSearch(event) {
+                    if (event !== undefined && typeof event.key !== "undefined") {
```

Comment:
`if (event instanceof KeyboardEvent)` would make the intent a lot clearer.

Reply:
ok